src/Coders/Model/Relations/BelongsToMany.php

```diff
@@ -77,11 +77,19 @@ public function hint()
      */
     public function name()
     {
-        if ($this->parent->usesSnakeAttributes()) {
-            return Str::snake(Str::plural(Str::singular($this->reference->getTable(true))));
+        if ($this->parent->shouldPluralizationTableName()) {
+            if ($this->parent->usesSnakeAttributes()) {
+                return Str::snake(Str::plural(Str::singular($this->reference->getTable(true))));
+            }
+
+            return Str::camel(Str::plural(Str::singular($this->reference->getTable(true))));
+        } else {
```

Comment:
Please, remove else statement. It's not needed since we are using returns inside ifs.